src/index.js

```diff
@@ -1,17 +1,21 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import * as serviceWorker from './serviceWorker'
+import GlobalStyles from './GlobalStyles'
+import { BrowserRouter as Router } from 'react-router-dom'
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <GlobalStyles />
+    <Router>
```

Comment:
Since your Navigation takes place in the App.js I don't think the Router here is necessary.

Reply:
I could move it into the App, but I think it is cleaner this way. That way I can use the Router anywhere in the App.